### PR TITLE
[DOC] date_range: Update broken image links

### PR DESCRIPTION
**Pull request by amh-mw**

### CODE REVIEW

**Review by lmignon — CHANGES_REQUESTED** [review]
Hi @amh-mw 

Thank you for your contrib. The `README.rst` file is generated from fragments defined into several files located under the `readme` directory. The right fix is to modify the links into these files. Once done, the `README.rst` will be generated at merge time. Can you update your PR accordingly?

**Comment by francesco-ooops** [comment]
@amh-mw you can take this as an example: https://github.com/OCA/server-ux/blob/16.0/template_content_swapper/readme/USAGE.rst?plain=1#L3

**Review by lmignon — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@lmignon I think it's better to use relative paths

File: date_range/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ inherit from `date.range.search.mixin`:
 
 This will make a `Period` field show up in the search view:
 
-  .. figure:: https://raw.githubusercontent.com/OCA/server-tools/12.0/date_range/static/description/date_range_many2one_search_field.png
+  .. figure:: https://raw.githubusercontent.com/OCA/server-ux/15.0/date_range/static/description/date_range_many2one_search_field.png
```
Comment (pedrobaeza) [inline date_range/readme/CONFIGURE.rst]: The correct way to put it and forget about this on each version is to put a relative path, and let the README generator to put the absolute one, but meanwhile, you see the correct image in local devel env.

```suggestion
  .. figure:: ../static/description/date_range_many2one_search_field.png
```

Comment (lmignon) [reply date_range/readme/CONFIGURE.rst]: @pedrobaeza you're right. I just checked the readme generator and relative paths are replaced by the right absolute path at generation.

**Review by lmignon — CHANGES_REQUESTED** [review]
As stated by others reviewers, the path must be relative

**Review by francesco-ooops — APPROVED** [review]
LGTM

@amh-mw would you be so kind to open a PR for v16 as well? Thanks!

**Review by lmignon — APPROVED** [review]

**Pull request merged**